src/test/java/edu/harvard/iq/dataverse/api/HarvestingClientsIT.java

```diff
@@ -54,6 +55,16 @@ public static void setUpClass() {
         setupCollection();
         
     }
+    @AfterEach
+    public void cleanup() {
```

Comment:
So, this will delete all the clients that the tests were leaving behind? - nice.

Reply:
Yes. It needs to be done after each test since there are now 2 harvesting tests using the same data.